AUTHORS.md

```diff
@@ -0,0 +1,9 @@
+All of the people who have made at least one contribution to versioned-hdf5.
+Authors are sorted by number of commits.
```

Comment:
Is this file generated automatically?

Reply:
I would sort this by last name.

Reply:
Yes, this is generated automatically.  I am not sure there is a sort-by-lastname option

Reply:
That would be an awesome addition to rever though, if you want to spend some time on it either of you